core/trino-main/src/main/java/io/trino/execution/scheduler/faulttolerant/EventDrivenFaultTolerantQueryScheduler.java

```diff
@@ -1913,6 +1905,14 @@ public Set<Map.Entry<ScheduledTask, PreSchedulingTaskContext>> listContextEntrie
                 return contexts.entrySet();
             }
 
+            private long getWaitingForNodeTasksCount(TaskExecutionClass executionClass)
```

Comment:
nit: getTasksWaitingForNodeCount

Reply:
It gets split into two methods later and `Tasks` is dropped from the name.

Reply:
Actually I did some renames. PTAL